lib/wallet.py

```diff
@@ -2878,13 +2885,16 @@ def is_multisig(self):
         return True
 
 
-wallet_types = ['standard', 'multisig', 'imported']
+
+#slptest_ wallet types to be renamed later before making available.  They are dev only/obscure wallet type.
```

Comment:
I'd suggest not merging anything until the actual names have been settled on. Likewise for the 'dev-only' switch above, these just say to me the code is still a work in progress.

Reply:
@markblundeberg the idea is be able to merge into mainline without maintaining a separate branch and without disrupting the main UX.  Maybe that is not best practice but it works and is low overhead.  Right now no one else is making SLP merge happen.